Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -970,63 +999,59 @@ - (void)addAttachmentUploadViewIfNecessary:(UIView *)attachmentView
 
     if (self.isOutgoing) {
         if (!self.attachmentStream.isUploaded) {
-            self.attachmentUploadView = [[AttachmentUploadView alloc] initWithAttachment:self.attachmentStream
-                                                                               superview:attachmentView
-                                                                 attachmentStateCallback:attachmentStateCallback];
+            __unused AttachmentUploadView *attachmentUploadView =
+                //            self.attachmentUploadView =
+                // This view will be added to attachmentView which will retain a strong reference to it.
+                [[AttachmentUploadView alloc] initWithAttachment:self.attachmentStream
+                                                       superview:attachmentView
+                                         attachmentStateCallback:attachmentStateCallback];
         }
     }
 }
 
-- (void)cropMediaViewToBubbbleShape:(UIView *)view
+- (void)showAttachmentErrorView:(UIView *)mediaView
```

Comment:
The `mediaView` isn't the error view right? So a better name could be: `showAttachmentErrorViewWithMediaView:(UIView *)mediaView`

Reply:
Done.